README.md

```diff
@@ -196,6 +193,9 @@ For more information about the scripts and their functionality, refer to the inl
 
 After completion of the bioinformatics analysis, the output file `UT-VH00770-230600_ncbi_submission_info.csv` can be used to create NCBI templates for biosample and SRA submission. Detailed instructions on submitting data to NCBI are located at `https://docs.google.com/document/d/1rGCWnDpGljdLqMs0FZ90TJLPtHRFpwIo-lalLKINn0w/edit?usp=sharing`. This file can only be accessed by anyone within UPHL. 
 
+# Debugging
+The scripts provide verbose logs throughout its operation. If there's an issue, the logs will often provide clues about what went wrong. Since 09/01/2023, I have been observing singularity/apptainer related issue when running the pipeline using `run_ww_analysis_auto.sh` at the viralrecon step. The current resolution is to resume and re-run the script using `-resume` flag, until we come up with a better solution to resolve this.
```

Comment:
Is there a step that viralrecon normally has issues with?

Reply:
Normally no. But I have a 'debugging' section in the README (Line 196) which states one of the issues that sometimes happens with viralrecon.

Reply:
The issue happens at 'snp_eff' step.